app/mailers/heya/campaign_mailer.rb

```diff
@@ -23,6 +23,12 @@ def build
       instance_variable_set(:"@#{user.model_name.element}", user)
       instance_variable_set(:@campaign_name, campaign_name)
 
+      headers_params = step.params.fetch("headers", nil)
+
+      if headers_params
+        headers headers_params
+      end
+
```

Comment:
```suggestion
      headers_params = step.params.fetch("headers", nil)
      headers(headers_params) if headers_params
```

Reply:
Thank you so much for the suggestion!  changed!